### PR TITLE
Update ome-faq.md

### PR DESCRIPTION
**Pull request by samschan-ms**
@KCCross we updated the message size limit together.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/ome-faq.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Microsoft 365 users can read and respond from Outlook for Windows and Mac (2013
   
 ## Is there a size limit for messages you can send with OME?
 
-Yes. The maximum message size you can send with OME, including attachments, is 30 MB.
+Yes. The maximum message size you can send with OME, including attachments, is 25 MB. For more information, see [Message limits](https://docs.microsoft.com/en-us/office365/servicedescriptions/exchange-online-service-description/exchange-online-limits#message-limits-1).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.md]:
```suggestion
Yes. The maximum message size you can send with OME, including attachments, is 25 MB. For more information, see [Message limits](https://docs.microsoft.com/office365/servicedescriptions/exchange-online-service-description/exchange-online-limits#message-limits-1).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please remove locale from the link.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks.

**Pull request merged**